gallery/src/styles/NxCard/NxCardHorizontalExample.tsx

```diff
@@ -0,0 +1,103 @@
+/*
+ * Copyright (c) 2019-present Sonatype, Inc.
+ * This program and the accompanying materials are made available under
+ * the terms of the Eclipse Public License 2.0 which accompanies this
+ * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
+ */
+import React from 'react';
+import { faShapes, faChartLine, faEdit, faTrash } from '@fortawesome/free-solid-svg-icons';
+
+import { NxFontAwesomeIcon, NxButton } from '@sonatype/react-shared-components';
+
+export default function NxCardHorizontalExample() {
+  return (
+    <div className="nx-card-container nx-card-container--column">
```

Comment:
At this point, the only thing this container effectively does is change the margin under the last card from 16px to 24px - which @dturner32 says is wrong anyway.  So I'm thinking we could just get rid of this container: if you want horizontal cards, you just use cards bare.  If you want vertical cards, you put them in the vertical card container.

Reply:
> At this point, the only thing this container effectively does is change the margin under the last card from 16px to 24px - which @dturner32 says is wrong anyway. So I'm thinking we could just get rid of this container: if you want horizontal cards, you just use cards bare. If you want vertical cards, you put them in the vertical card container.

It does more than that, it tells the cards within it whether they are vertically or horizontally oriented. I'm looking towards being able to toggle between the two states if desired.